src/acquisition/covidcast/csv_importer.py

```diff
@@ -221,27 +221,30 @@ def validate_missing_code(row, attr_quantity, attr_name):
     to infer missing codes except for a very simple cases; the default
     is to produce an error so that the issue can be fixed in indicators.
     """
-    if hasattr(row, "missing_" + attr_name):
-      missing_entry = getattr(row, "missing_" + attr_name)
-      try:
-        missing_entry = int(missing_entry)
-      except ValueError:
-        return None
-      # A missing code should never contradict the quantity being present,
-      # since that will be filtered in the export_to_csv util in
-      # covidcast-indicators; nonetheless this code is here for safety.
-      if attr_quantity is not None and missing_entry != Nans.NOT_MISSING.value:
-        return None
-      elif attr_quantity is None and missing_entry == Nans.NOT_MISSING.value:
-        return None
-      return missing_entry
-    else:
-      if attr_quantity is None:
-        return Nans.OTHER.value
+    logger = get_structured_logger('load_csv') if logger is None else logger
+    missing_entry = getattr(row, "missing_" + attr_name, None)
+
+    try:
+      missing_entry = CsvImporter.floaty_int(missing_entry) # convert from string to float to int
```

Comment:
good ol' floaty_int

Reply:
that ol' floaty_int we all know and love